tests/flow/test_memory_usage.py

```
@@ -0,0 +1,294 @@
+            pass
+
+        # create graph
+        self.graph.query("RETURN 1")
+
+        # missing samples count
+        cmd = f"GRAPH.MEMORY USAGE {GRAPH_ID} SAMPLES"
+
+        try:
+            res = self.conn.execute_command(cmd)
+            self.env.assertTrue(False)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)